dts/bindings/spi/nxp,imx-xspi.yaml

```
@@ -0,0 +1,20 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP XSPI controller
+
+compatible: "nxp,imx-xspi"
```

```suggestion
compatible: "nxp,xspi"
```

Avoiding family names in compatibles is best because we may reuse this IP on non-IMX.RT cores

---

Updated.